LSP-Grammarly.sublime-commands

```diff
@@ -0,0 +1,13 @@
+[
+    { "caption": "LSP-Grammarly: Login to Grammarly", "command": "lsp_grammarly_execute_login"},
+    { "caption": "LSP-Grammarly: Logout from Grammarly", "command": "lsp_grammarly_execute_logout"},
+    { "caption": "LSP-Grammarly: Check if connected to Grammarly", "command": "lsp_grammarly_execute_is_connected"},
```

Comment:
I think this should say: `Check if logged-in to Grammarly` because it really seems to be about login state and not connected state (which technically can be true even without logging-in).